src/Mjolnir.ts

```diff
@@ -343,13 +343,12 @@ export class Mjolnir {
         if (!additionalRoomIds) additionalRoomIds = [];
         if (!Array.isArray(additionalRoomIds)) additionalRoomIds = [additionalRoomIds];
 
-        if (config.RUNTIME.client && (config.verboseLogging || LogLevel.INFO.includes(level))) {
```

Comment:
Wasn't this done likt this to make sure that mjolnir still correctly logs even when the Matrix Client fails to initialize?

Reply:
That's here https://github.com/matrix-org/mjolnir/blob/main/src/index.ts#L45-L69

Reply:
Ah alright, then this can be closed